src/NoteMouseHandler/SelectionMouseHandler.js

```diff
@@ -1,72 +1,71 @@
 import _ from "lodash"
-import MouseHandler, { defaultActionFactory, getLocal } from "./NoteMouseHandler"
+import MouseHandler, { defaultActionFactory } from "./NoteMouseHandler"
```

Comment:
__[jshint]__
*  'import' is only available in ES6 (use 'esversion

<!-- Comment by [SideCI](https://sideci.com) -->